.github/workflows/test.yml

```diff
@@ -1,18 +1,55 @@
+name: Test and build outline-cli
+
 on: [push, pull_request]
-name: Test
+
 jobs:
   test:
+    name: Test
     strategy:
       matrix:
         go-version: [1.19.x]
         os: [ubuntu-latest]
     runs-on: ${{ matrix.os }}
+
     steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
       - name: Install Go
         uses: actions/setup-go@v4
         with:
           go-version: ${{ matrix.go-version }}
+
+      - name: Get dependencies
+        run: go mod download
```

Comment:
Why should we do this? Go test will download it automatically, no? 🤔

Reply:
I think this just makes our intentions and the stuff that happens "explicit". We can remove it if you like.

Reply:
This is not necessary